src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -9,7 +9,12 @@
 @inject IStringLocalizer<Dashboard.Resources.TraceDetail> Loc
 @inject IStringLocalizer<ControlsStrings> ControlStringsLoc
 
-<PageTitle><ApplicationName ResourceName="@nameof(Dashboard.Resources.TraceDetail.TraceDetailPageTitle)" Loc="@Loc" /></PageTitle>
+<PageTitle>
+    <ApplicationName
+        AdditionalText="@(_trace?.RootSpan?.Name)"
```

Comment:
On line 32 there is a header displayed for the page. Move creating that title to a method and reuse here and there.